src/pages/index.js

```diff
@@ -1,27 +1,31 @@
 import Link from "next/link";
 
-export default function TransactionList({ initialData}) {
+export default function TransactionList({ transactions }) {
   // Sort transactions by date in descending order
-  const sortedTransactions = initialData.sort(
-    (a, b) => new Date(b.date) - new Date(a.date)
+  const sortedTransactions = transactions.sort(
+    (a, b) => new Date(b.date) - new Date(a.date),
   );
 
   return (
     <div>
```

Comment:
Do you use ```<div>```for any styling reason? Otherwise just use empty fragments ```<> </>```

Reply:
will be changed in the styling ticket